cli/cmdlineparser.cpp

```diff
@@ -925,7 +925,7 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
         mSettings->checkAllConfigurations = true;
 
     if (mSettings->force)
-        mSettings->maxConfigs = ~0U;
+        mSettings->maxConfigs = static_cast<int>(~0U);
```

Comment:
that cast changes the sign. we should either have a large positive value or -1 explicitly. Probably a large positive value is wanted..

Reply:
This will be done implicitly anyways - I just doing it explicitly so UBSAN know it is actually intended.

Reply:
actually this was a "true positive" in my opinion so we should not just hide it. I'd rather either write a large positive value or -1 (to keep old behavior).. and I have the feeling that a large positive value would be preferable.

Reply:
I changed this one with e7c462b8279428e9722e0d1a1b85b9d84308be2b